assets/demos/demoapp-pipeline.yml

```diff
@@ -0,0 +1,37 @@
+---
+# yamllint disable line-length
+name: "Publish and test demo app"
+on:
+  push:
+    branches: [main]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare required variables
+        shell: pwsh
+        run: |
+          # Set the ENVIRONMENT_SUFFIX to a random value
+          # This will be used to create a unique resource group name
+          $env:ENVIRONMENT_SUFFIX = (Get-Random).ToString("x8")
+
+          # Set the location to westus2
+          $env:LOCATION = "westus2"
+
+      - name: Deploy infrastructure and demo app
+        shell: pwsh
+        run: |
+          ./deploy-demoapp.ps1
```

Comment:
Where are doing the az login in the workflow? Can we verify that this pipeline is functioning and provide a test execution?